pkg/virt-api/api.go

```diff
@@ -952,10 +953,11 @@ func (app *virtAPIApp) Run() {
 
 	stopChan := make(chan struct{}, 1)
 	defer close(stopChan)
-	go webhookInformers.VMIInformer.Run(stopChan)
```

Comment:
maybe I'm missing something obvious, but why is this informer not started anymore?

Reply:
Because it is a left-over. We don't use it in virt-api. Updated the PR description.

Reply:
ah, its use in the webhook was removed with a recent PR, didn't have that locally yet, sorry.